### DISCUSSION TITLE
BEAKL p_rn

### DISCUSSION THREAD

**Discussion by boogerlad** [precondition/precondition.github.io#15] [Q&A] [answered]
Under `assets/images/BEAKL_P_RN-KLE.png`, what are these keys?
![image](https://user-images.githubusercontent.com/1903750/116190143-16d39f80-a6f8-11eb-8f25-1a64791ac7a8.png)

Also, has anyone tried this layout? I think this layout uses the most character keys for thumbs.

**Comment by precondition** [comment, marked as answer]
<kbd>R⇧</kbd> and <kbd>L⇧</kbd> mean respectively Right and Left Shift.
<kbd>␣</kbd> is the symbol for the space key. The circle in the middle is an indicator of homing/resting position for the fingers.
<kbd>⌫</kbd> is the backspace key and <kbd>⌦</kbd> is the delete key.

I'm not aware of any real users of this layout but its many thumb letter keys warrant its place in the list.

**Reply by boogerlad** [reply]
Are right and left shift applied simultaneously when pressed? What are the benefits over just right xor left?
With the backspace and delete key, how does that work? Are they on different layers? Simultaneous activation doesn't make sense.

**Reply by precondition** [reply]
I don't know why the original BEAKL diagrams felt the need to draw left *and* right shift but you should treat it as a simple shift key.

The legends in the lower right corner are the keysyms on a layer. Naturally, they don't get triggered together. That would have meant that pressing the <kbd>A</kbd> key would produce "a0" all the time which is obviously not the intended behaviour.

**Reply by iandoug** [reply]
There are two shifts because Keyboard Layout Analyzer needs two shifts. So we put them on one key (typically thumb) and all is well.

lower right legend: accessed with AltGr.
upper right legend: accessed with shift-AltGr.

Cheers, Ian
